src/scripts/towns/TownList.ts

```diff
@@ -4102,7 +4102,7 @@ const CoumarineBirdwatcher = new NPC('Birdwatcher', [
 const CoumarineElectricTrainer = new NPC('Electric Trainer', [
     'My Ampharos has gotten a boost since a friendly soul gave me an Ampharosite!',
     'Luckily my Ampharos was already strong enough to use it!',
-    'If you find that fella, tell him the code <b>ELECTRIC-GIRAFFE</b> and he might help you too!',
+    'If you find that fella, tell him the code <b>DO-MAGEARNA-DREAM-OF-MAREEP?</b> and he might help you too!',
```

Comment:
why the sudden all caps kebab when old codes don't use '-' ?
![image](https://user-images.githubusercontent.com/25463975/212367425-f54f6b9a-261d-4474-9652-88d1904137b2.png)

Reply:
Dunno. That's how it already was

Reply:
It would be easy if the codes where predictable

Reply:
fair enough, i did blind try that exact code without the '-' back in 2022 xD

Reply:
(serious answer) It would maybe be more consistent if we did it the same way, but i think the joke is easier to read with the `-`, so i added them. But yeah, we can always change them later, if we think it's too weird